src/main/java/com/spotify/reaper/cassandra/JmxConnectionFactory.java

```diff
@@ -17,6 +17,7 @@
 
 import com.datastax.driver.core.policies.EC2MultiRegionAddressTranslator;
 import com.spotify.reaper.ReaperApplicationConfiguration.JmxCredentials;
+import com.spotify.reaper.ReaperApplication;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.spotify.reaper.ReaperApplication'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)